knotx-adapter/knotx-adapter-common/src/main/java/io/knotx/adapter/common/http/HttpClientFacade.java

```diff
@@ -143,7 +143,7 @@ protected ClientRequest buildServiceRequest(ClientRequest originalRequest, JsonO
       filteredHeaders.names().forEach(
           headerName -> httpRequest.putHeader(headerName, filteredHeaders.get(headerName)));
       if (!serviceRequest.getFormAttributes().isEmpty()) {
-        httpRequest.end(FormBodyBuilder.createBody(serviceRequest.getFormAttributes()));
+        httpRequest.end(UrlEncodedBodyBuilder.encodeBody(serviceRequest.getFormAttributes()));
```

Comment:
We should check 'Content-Type' header and based on this value decide how to build the body. Default should be 'application/x-www-form-urlencoded'.